packages/uicore/src/components/TableV2/TableV2.tsx

```diff
@@ -195,7 +195,11 @@ export const TableV2 = <Data extends Record<string, any>>(props: TableProps<Data
           )
         })}
       </div>
-      {pagination ? <Pagination className={css.pagination} {...pagination} /> : null}
+      {pagination ? (
```

Comment:
can we just combine it into one conditon like 
```
pagination && pagination.itemCount > pagination.pageSize
```
and return null if it's false.